components/mpas-seaice/cime_config/buildnml

```diff
@@ -199,12 +199,12 @@ def buildnml(case, caseroot, compname):
         decomp_date += '180820'
         decomp_prefix += 'mpas-cice.graph.info.'
     elif ice_grid == 'EC30to60E2r2':
-        grid_date += '200908'
+        grid_date += '210210'
         grid_prefix += 'seaice.EC30to60E2r2'
         decomp_date += '200908'
         decomp_prefix += 'mpas-seaice.graph.info.'
     elif ice_grid == 'EC30to60E2r2_ICG':
-        grid_date += '201001'
+        grid_date += '210210'
```

Comment:
@vanroekel - I couldn't find this file on the inputdata server

Reply:
I accidentally named this `-1900` it is fixed now